distribution/utils/metrics.go

```diff
@@ -0,0 +1,39 @@
+package utils // import "github.com/docker/docker/distribution/utils"
+
+import (
+	"io"
+
+	gometrics "github.com/docker/go-metrics"
+)
+
+var (
+	metricsNS = gometrics.NewNamespace("engine", "daemon", nil)
+
+	// ImagePullBytes is a running tally of the bytes downloaded and extracted
+	// by image pull operations. By taking the derrivative of this metric over
+	// time, image pull throughput can be measured.
+	ImagePullBytes = metricsNS.NewCounter("image_pull_bytes", "The number of bytes processed as part of image pulls")
+)
```

Comment:
The metric definition seems out of place to me here. Move it to distribution/pull_v2.go and make it package-private?